0.data_analysis/README.md

```diff
@@ -0,0 +1,6 @@
+# 0.data_analysis
+This module focuses on exploring relationships in cell morphology data not included in the [nf1_cellpainting_data](https://github.com/WayScience/nf1_cellpainting_data) repository.
```

Comment:
> Recommend instead of saying that this is "not included" in the other repo, it might be more helpful to link to the `4.analyze_data` module and state what analysis was perform that is different from this repo (e.g., UMAPs, linear models, etc.).

Maybe we could list some examples, and then refer these readers to both modules READMEs' for more information by providing links? I think this would help keep the READMEs more DRY.
> We might also consider renaming one of these folders in one repo, either this one or the other one. This doesn't have to be done in this PR, but it will be good to discuss once we are ready to preprint the manuscript.

Sounds good, we can do that. I think we will also want to rename the analysis repo

Reply:
> Maybe we could list some examples, and then refer these readers to both modules READMEs' for more information by providing links? I think this would help keep the READMEs more DRY.

I can see that. I recommend not mentioning the image profiling repo at all here then and just focus on discussing what is going on in that module.

Reply:
That works, I'll make the change